nvme_gw_server.py

```diff
@@ -273,6 +282,17 @@ def nvmf_get_subsystems(self, request, context):
 
         return pb2.subsystems_info(subsystems=json.dumps(ret))
 
+    def ping(self):
+        try:
+            ret = self.spdk_rpc.rpc_get_methods(
```

Comment:
Probably not measurable, but might something like `log_get_level` be cheaper here?

Reply:
I  have no a good idea. I am fine with `log_get_level` too.

Reply:
I decided to use `spdk_get_version`, which I suppose is also "cheap".